anta/tests/configuration.py

```diff
@@ -131,3 +131,42 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure("Following patterns were not found: " + ",".join(failure_msgs))
+
+
+class VerifyMcsClientMounts(AntaTest):
+    """Verifies the MCS Clients mount status.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the MCS mount status on MCS Clients are mountStateMountComplete.
+    * Failure: The test will fail if the MCS mount status on MCS Clients are not mountStateMountComplete.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.configuration:
```

Comment:
is it really a configuration test?

Would you not prefer a new MCS category maybe?

cc @carl-baillargeon

Reply:
+1 for having a new category and put the CVX stuff in there.

Reply:
let's do it

Reply:
what would be the new category be? CVXConfiguration ? I do consider it as part of "device configuration"

Reply:
You can create a new module --> `anta/tests/cvx.py`